crypto/fipsmodule/ec/p384.c

```diff
@@ -685,10 +689,10 @@ OPENSSL_STATIC_ASSERT(P384_MUL_WSIZE == 5,
 // It forces an odd scalar and outputs digits in
 // {\pm 1, \pm 3, \pm 5, \pm 7, \pm 9, ...}
 // i.e. signed odd digits with _no zeroes_ -- that makes it "regular".
-static void p384_felem_mul_scalar_rwnaf(int16_t *out, const unsigned char *in) {
+static void p384_felem_mul_scalar_rwnaf(int16_t *out, const EC_SCALAR *in) {
   int16_t window, d;
 
-  window = (in[0] & P384_MUL_WSIZE_MASK) | 1;
+  window = (in->words[0] & P384_MUL_WSIZE_MASK) | 1;
```

Comment:
I don't like that we are mixing `int16_t` with `uint64_t` here but I don't have a better suggestion.